src/pykeen/pipeline.py

```diff
@@ -525,7 +525,7 @@ def save_to_s3(self, directory: str, bucket: str, s3=None) -> None:
 
 
 def _reduce_embeddings(embeddings, reducer, fit: bool = False):
-    embeddings_numpy = embeddings.weight.detach().numpy()
+    embeddings_numpy = embeddings.forward(None).detach().numpy()
```

Comment:
two comments:
* directly use `__call__` instead of explicitly calling `forward()`
* Make sure to use `.cpu()` before calling `numpy()`, in case the tensor resides on a GPU. (Otherwise leads to `TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.`)

In total: `embeddings(None).detach().cpu().numpy()`

Reply:
oops these are good suggestions... sorry I didn't wait

Reply:
Is the default index None? Then what would you think about writing it like `embeddings().detach().cpu().numpy()`

Reply:
Counterpoint, making this explicit protects in case the default changes

Reply:
:+1: for making None the default (if it is not yet). I cannot think of another useful default besides returning all representations.

Reply:
I checked - it is. We can come back to this after the new model structuring branch is done becuase I don't want to get a diff on this again